src/components/LandPage.js

```diff
@@ -0,0 +1,75 @@
+import {
+  HeaderStyle,
+  LandPageStyle,
+  SearchBoxStyle,
+} from "../styles/LandPage.styled";
+import { useContext, useEffect } from "react";
+import Context from "./context";
+import MoviesContainer from "./MoviesContainer";
+import NoResult from "./NoResult";
+import { useNavigate } from "react-router-dom";
+
+export const LandPage = () => {
+  let navigate = useNavigate();
+  const a = useContext(Context);
+
+  const API_KEY = "api_key=0decbb49ed9b3d0c6017d7721a14c106";
+  const BASE_URL = "https://api.themoviedb.org/3";
+  const API_URL =
+    BASE_URL + "/discover/movie?sort_by=popularity.desc&" + API_KEY;
+  const searchURL = BASE_URL + "/search/movie?" + API_KEY + "&query=";
+
+  async function getMovie(url) {
```

Comment:
Make use of advanced features of ES6 and above. Here you can use arrow function.